src/lib/db/schema.ts

```diff
@@ -284,6 +284,7 @@ export const playerGameSessionChatHistoryItems = pgTable(
       .references(() => playerGameSessions.id, {
         onDelete: "cascade",
       }),
+    submitted: boolean("submitted").notNull().default(false),
```

Comment:
can you add a new `content` type instead of this. that way we can also capture when people run tests